content/rendering-with-batch/monitor/optional_fis.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Optional - Spot Interruption"
+date: 2022-09-20T00:00:00Z
+weight: 145
+---
+
+
+## This step is optional and is not required
+
+{{% notice warning %}}
+You should wait until the AWS Batch job from your first experiment has fully completed before starting this optional lab.
+{{% /notice %}}
+
+To experiment with how AWS Batch jobs handles EC2 Spot Interruptions, you can follow this guide...
```

Comment:
Rephrase line 14 to be more descriptive, move it to the top of the page and replace EC2 Spot Interruptions by Spot Instance interruptions.